src/misc/lv_timer.h

```diff
@@ -26,6 +26,23 @@ extern "C" {
 
 #define LV_NO_TIMER_READY 0xFFFFFFFF
 
+/**
+ * Call it in the super-loop of main() or threads. It will run lv_timer_handler()
+ * with a given period in ms.
+ * This macro is used to simplify the porting.
+ * @param __ms the period for running lv_timer_handler()
+ */
+#define lv_run_timer_handler_in_period(__ms)                                    \
```

Comment:
What about this name?
```suggestion
#define lv_timer_handler_run_in_period(__ms) 
```

Reply:
Better!!!